kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Other imports
+import os
+
+class TimeBasedAsciiFileWriterUtility(object):
+    """This utility handles a file to which results are to be written.
```

Comment:
This is an improvement but I still think the class could be reduced. Now it appears to provide a wrapper to an opened file at the position ready to write output data. After the constructor returns it basically behaves like a native python file object (with limited functionality). Why not just return the naked file to the user rather than a wrapper class? The functions `write`, `flush` and `close` are redundant and could be eliminated.